src/main/java/org/ibp/api/rest/breedingmethod/BreedingMethodResourceGroup.java

```diff
@@ -67,4 +66,18 @@
 	) {
 		return new ResponseEntity<>(MethodGroup.getAll(), HttpStatus.OK);
 	}
+
+	@ApiOperation(value = "List breeding method filtered by favorites")
+	@RequestMapping(value = "/crops/{cropName}/breedingmethod-favorites", method = RequestMethod.GET)
```

Comment:
I think we can just renamed to breedingmethods given that favorites is just a filter